benchkit/hdc/__init__.py

```diff
@@ -0,0 +1,219 @@
+# SPDX-License-Identifier: MIT
+"""
+Module to handle hdc (harmony debug bridge) interactions between host and remote phone.
```

Comment:
It seems to stand for `OpenHarmony Device Connector` 
Lets also add a link to the official documentation: https://docs.openharmony.cn/pages/v5.0/en/application-dev/dfx/hdc.md